libc/src/__support/UInt.h

```diff
@@ -27,6 +27,11 @@ namespace LIBC_NAMESPACE::cpp {
 
 template <size_t Bits, bool Signed> struct BigInt {
 
+  // This being hardcoded as 64 is okay because we're using uint64_t as our
```

Comment:
Can we also add
```
using word_type = uint64_t;
LIBC_INLINE_VAR static constexpr size_t WORD_SIZE = sizeof(word_type) * CHAR_T;
...
cpp::array<word_type, WORDCOUNT> val{};
```

Reply:
I've done that, though I'm leaving the large scale refactoring for a followup patch.